src/logic/calculate.js

```diff
@@ -0,0 +1,12 @@
+import * as math from 'mathjs';
+
+function calculateExpression(expression) {
+  try {
+    const calculatedResult = math.evaluate(expression);
+    return calculatedResult.toString();
+  } catch (error) {
+    return 'Error';
+  }
+}
+
+export default calculateExpression;
```

Comment:
Kindly make sure that you are using the calculate.js file that is provided in the project requirements.

here is the [link](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/code-samples/calculate.js)